lib/devcycle-ruby-server-sdk/localbucketing/bucketed_user_config.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+
+module DevCycle
+  class Environment
+    attr_accessor :id
+    attr_accessor :key
+  end
+
+  class FeatureVariationMap
+    attr_accessor :the_6216422850294_da359385_e8_b
```

Comment:
wat

Reply:
Autogenerated stuff - need to clean this up once we start implementing the SDK to see whether or not this is even used